src/main/java/mate/jdbc/service/DriverServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver get(Long id) {
+        return driverDao.get(id).orElseThrow(() ->
+                new NoSuchElementException("Can't get user with id" + id));
```

Comment:
```suggestion
                new NoSuchElementException("Can't get driver with id" + id));
```

Reply:
i have new line in the end of src/main/resources/init_db.sql
https://prnt.sc/1tx9mfz